WordPress/Classes/Utility/Logging/EventLoggingDelegate.swift

```diff
@@ -1,10 +1,15 @@
 import Foundation
 import AutomatticTracks
 import AutomatticEncryptedLogs
+import BuildSettingsKit
 
 struct EventLoggingDelegate: AutomatticEncryptedLogs.EventLoggingDelegate {
 
     var shouldUploadLogFiles: Bool {
+        guard BuildSettingsEnvironment.current != .test else {
+            return false
+        }
```

Comment:
After migrating to reading secrets from `BuildSecrets`, unit tests
    started crashing in the background thread attempting the encrypted logs
    upload.

    This is possibly related to the changes in how the secrets are accessed,
    but anyways we should not upload logs from the unit tests as that's just
    noise.

Reply:
Nice! It also used  to crash for OSS setup.